ci-operator/jobs/open-cluster-management/submariner-addon/open-cluster-management-submariner-addon-main-postsubmits.yaml

```diff
@@ -0,0 +1,93 @@
+postsubmits:
+  open-cluster-management/submariner-addon:
+  - agent: kubernetes
+    branches:
+    - ^main$
+    cluster: build02
+    decorate: true
+    extra_refs:
+    - base_ref: main
```

Comment:
are you sure it should be main? release repo does not have main branch

Reply:
fixed